openmdao/devtools/make_hohmann_figures.py

```diff
@@ -0,0 +1,293 @@
+import os.path
```

Comment:
agree it's a good question...   my suggestion would be that, in the same way that the test for this example comes in under the examples directory, the other files that support the example in the docs would also belong there, rather than being split into a different location in the repo.  Just my thoughts.